docs/src/content/docs/core/define-integration.mdx

```diff
@@ -148,4 +148,27 @@ In setup, you'll want to add any logic that is shared between any hook, for exam
 - Calling `createResolver`
 - Save the `config` from `astro:config:done` to be used in later hooks
 
-It needs to return Astro hooks. 
\ No newline at end of file
+It needs to return Astro hooks.
+
+## Using non-built-in Astro hooks
+
+Some official Astro tools, like [`@astrojs/db`](https://docs.astro.build/en/guides/integrations-guide/db/), define new integration hooks that are not built into Astro core. AIK, by default, don't integrate with those hooks.
+
```

Comment:
Might be worth having a heading here so we can easily link people to the specific `db` section.

`### @astrojs/db`